Source/DafnyLanguageServer/Workspace/DocumentManager.cs

```diff
@@ -127,6 +133,40 @@ public class DocumentManager {
     Compilation.Start();
   }
 
+  private static DafnyOptions DetermineDocumentOptions(DafnyOptions serverOptions, DocumentUri uri) {
+    ProjectFile? projectFile = null;
+
+    var folder = Path.GetDirectoryName(uri.GetFileSystemPath());
+    while (!string.IsNullOrEmpty(folder)) {
+      var children = Directory.GetFiles(folder, "dafny.toml");
```

Comment:
I don't think that's a good idea, then you can't have non-Dafny `*.toml` files in your filetree above a `.dfy` file, because the IDE would recognize them as Dafny project files.

We could allow any `*dafny.toml`, but that seems needlessly complicated. Is there a problem with only accepting `dafny.toml` ? If you want to have different project files that use different source files, there shouldn't be an issue in putting those in a directory as well. It's similar to TypeScript: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

Reply:
Anyways, if we leave it like it is now, we can always allow more later ;-)